src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -41,6 +41,8 @@ class Definition
     private $synchronized = false;
     private $lazy = false;
     private $decoratedService;
+    private $autowired = false;
+    private $autowiringTypes = array();
```

Comment:
you might want change this prop. name with `autowire`according to @weaverryan suggestion